.gitignore

```diff
@@ -2,6 +2,7 @@
 .vscode
 .idea
 .DS_Store
+.cache
```

Comment:
This is duplicated from PR #353, please remove it from this PR or close the other PR